React/Modules/RCTAnimationGeometry.h

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+double RemapValue(double value, double low1, double high1, double low2, double high2 );
```

Comment:
This should be a static function, since it's private. Note: if you make it static, there's no need to pre-declare it here.